draft-ietf-tls-esni.md

```diff
@@ -1159,8 +1194,6 @@ for ExtensionType (defined in {{!RFC8446}}):
 
 1. encrypted_client_hello(0xff08), with "TLS 1.3" column values being set to
    "CH, EE", and "Recommended" column being set to "Yes".
-2. ech_nonce(0xff03), with the "TLS 1.3" column values being set to "CH", and
-   "Recommended" column being set to "Yes".
 3. outer_extension(0xff04), with the "TLS 1.3" column values being set to "CH",
```

Comment:
Let's move this to 0xff04.

Reply:
What should we do about "outer_extension" (currently 0xff04)?

Reply:
Oops, I meant 0xff03. Heh. Let's just revisit these values in a followup PR.